_sections/30-bindings.md

```diff
@@ -227,6 +228,7 @@ The following meta elements are supported:
 | element      | description                                       | default datatype | value             | namespace
 |--------------|---------------------------------------------------|----------------------------------------------------------------------------
 | `instanceID` | The unique ID of the record [required]            | string           | concatenation of 'uuid:' and uuid() | same as meta block
+| `instanceName` | The name of the form. If no binding is specified, defaults to the form title.             | string           |  | same as meta block
```

Comment:
Thanks so much for making this addition! How about saying "The name of the record" to match `instanceID` above?

I think `If no binding is specified, defaults to the form title.` should be removed because from a spec perspective, if there's no binding, nothing at all is included. Different consumers of the spec can and do make different decisions about what to display if no `instanceName` is provided. ODK Collect does show the form title but ODK Central shows the `instanceID`, for example.

Reply:
I have pushed the correction!